<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listeconsulter</class>
 <widget class="QDialog" name="listeconsulter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background:#001328;
</string>
  </property>
  <widget class="QTableView" name="tableViewed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>701</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;
color:black;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:white;</string>
   </property>
   <property name="title">
    <string>Filtre</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="rdodate">
       <property name="styleSheet">
        <string notr="true">COLOR:white;</string>
       </property>
       <property name="text">
        <string>Entre 2 Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="styleSheet">
        <string notr="true">background:white;
color:black;</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="date">
        <date>
         <year>2022</year>
         <month>10</month>
         <day>11</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="styleSheet">
        <string notr="true">background:white;
color:black;</string>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="date">
        <date>
         <year>2022</year>
         <month>10</month>
         <day>11</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="rdomois">
       <property name="styleSheet">
        <string notr="true">COLOR:white;</string>
       </property>
       <property name="text">
        <string>Mois</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtValue_3">
       <property name="styleSheet">
        <string notr="true">background:white;
color:black;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="rdoannee">
       <property name="styleSheet">
        <string notr="true">COLOR:white;</string>
       </property>
       <property name="text">
        <string>Année</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtValue_2">
       <property name="styleSheet">
        <string notr="true">background:white;
color:black;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>500</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>Nombre total de jours:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_liste">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>560</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">COLOR:white;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>150</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblNommed">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>Nom:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtNommed">
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>70</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>N°Patient       </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtValue">
      <property name="styleSheet">
       <string notr="true">background:white;
color:black;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">background-color: #5864A4;
color:white;</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size:15px;</string>
   </property>
   <property name="text">
    <string>LISTE DES MEDECINS CONSULTES PAR UN PATIENT</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>540</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string>Salaire Total:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">COLOR:white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
